CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # TRILLIAN Changelog
 
+## v1.0.9 - CloudSpanner Improvements & Internal API changes etc.
+
+Published 2018-04-17 07:01:46 +0000 UTC
+
+Changes are in progress (e.g. see PR #1037) to rework the internal signed root format used by the log RPC server to be more useful / interoperable. Currently they are mostly internal API changes to the log and map servers.

```

Comment:
PR #1037 isn't auto linking -- could we make it `[#1037](https://github.com/google/trillian/pull/1037)` ?

Reply:
Done.